src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -60,9 +60,14 @@
         PREVIEW
     }
 
+    interface Type {
+//        String ARTICLE = "http://www.ft.com/ontology/content/Article";
```

Comment:
I feel that currently is not clear enough how this type will be used other than ContentPackage. What defines an Article and what defines a Content? I'd rather vote for deleting the fields entirely, but I'd like to hear more opinions